ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/services/blob-video-storage.service.spec.ts

```diff
@@ -1,30 +1,35 @@
 import { BlobVideoStorageService } from './blob-video-storage.service';
 import { BlobStorageService } from './blob-storage.service';
+import { DeviceType } from './device-type';
 import { Config } from '../../shared/models/config';
+import { VideoFiles } from './video-files';
 
 describe('BlobVideoStorageService', () => {
 
-    it('should return the videoname for in-hearing video', () => {
-        const config = new Config();
-        config.baseVideoUrl = 'http://test.com';
-        const blobVideoStorageService = new BlobVideoStorageService(new BlobStorageService(config));
-        const actualValue = blobVideoStorageService.inHearingExampleVideo;
-        expect(actualValue).toEqual('http://test.com/btd_individual_laptop_large.mp4');
-    });
+  it('should return the video name for in-hearing video', () => {
```

Comment:
not sure we need to test each and every value, if we do, we could consider a method that tests each value of the enumeration. consider:

```typescript
it('should return a video url for each video', () => {
  foreach (const key of Object.keys(VideoUrls)) {
    expect(blobVideoStorageService.getVideoFileUrl(key)).not.toBeNull();
  }
});
```

Reply:
Done